sql/internal/specutil/convert.go

```diff
@@ -457,3 +472,25 @@ func colRef(cName string, tName string) *schemaspec.Ref {
 func schemaRef(n string) *schemaspec.Ref {
 	return &schemaspec.Ref{V: "$schema." + n}
 }
+
+// convertCommentFromSpec converts a spec comment attribute to a schema element attribute.
+func convertCommentFromSpec(spec interface {
+	Attr(string) (*schemaspec.Attr, bool)
+}, attrs *[]schema.Attr) error {
```

Comment:
```suggestion
func convertCommentFromSpec(spec interface {Attr(string) (*schemaspec.Attr, bool) }, attrs *[]schema.Attr) error {
```

Not that long for adding this line break (I'm not fan of it tbh)

Reply:
`gfmt` thinks otherwise. `gofmt` is king :man_shrugging: